Dockerfile

```diff
@@ -36,6 +37,8 @@ RUN npm run build:production
 
 # Prepare the dist image with additional node_modules
 WORKDIR /ghostfolio/dist/apps/api
+# Exclude unnecessary dev deps
+RUN sed -i '/@nx/d' package.json
```

Comment:
Could we use `npm install --omit=dev` to prevent the installation of `devDependencies`?
Source: https://stackoverflow.com/a/9276112/744012

Reply:
@dtslvr in this case the generated package.json includes `@nx/webpack` in the production dependencies so I think the `--omit=dev` won't prevent it from being installed

Reply:
@dtslvr I excluded the webpack file in the nx project options in https://github.com/ghostfolio/ghostfolio/pull/3642/commits/12e4eb5b9ea64e0e93a7c11ea15be63c5ebfa63e and it seemed to fix the problem